ffmpegdecoder.cpp: In member function ‘void Gejengel::FFmpegDecoder::initializeAudio()’:
ffmpegdecoder.cpp:107:64: error: ‘CODEC_TYPE_AUDIO’ was not declared in this scope

--- origsrc/gejengel-0.1.4/src/AudioDecoder/ffmpegdecoder.cpp	2010-03-19 09:31:43.000000000 -0500
+++ src/gejengel-0.1.4/src/AudioDecoder/ffmpegdecoder.cpp	2012-02-12 19:12:54.132673800 -0600
@@ -104,7 +104,7 @@ void FFmpegDecoder::initializeAudio()
 {
     for(uint32_t i = 0; i < m_pFormatContext->nb_streams; ++i)
     {
-        if (m_pFormatContext->streams[i]->codec->codec_type == CODEC_TYPE_AUDIO)
+        if (m_pFormatContext->streams[i]->codec->codec_type == AVMEDIA_TYPE_AUDIO)
         {
             m_pAudioStream = m_pFormatContext->streams[i];
             m_AudioStream = i;
